eureka-client/src/main/java/com/netflix/discovery/EurekaClient.java

```diff
@@ -94,23 +94,31 @@
     public InstanceInfo.InstanceStatus getInstanceRemoteStatus();
 
     /**
+     * @deprecated see {@link com.netflix.discovery.endpoint.EndpointUtils} for replacement
+     *
      * Get the list of all eureka service urls for the eureka client to talk to.
      *
      * @param zone the zone in which the client resides
      * @return The list of all eureka service urls for the eureka client to talk to.
      */
+    @Deprecated
```

Comment:
The replacement is not one to one, as EndpointUtils expects more arguments. Shall we create a new interface  for zone management? I am working now on transport refactoring, where I have introduced `ClusterResolver` abstraction. Zone analysis is done actually because of the transport needs, so probably lots of this logic will be used there.